Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
+------------------
+
+.. data:: CLOCK_HIGHRES
+
+   The Solaris OS has a CLOCK_HIGHRES timer that attempts to use an optimal
```

Comment:
That's weird. I don't know why it didn't work. Could you try copy and paste the backticks in line 27 in Doc/library/time.rst? Perhaps something was wrong with my earlier comment.